backend/utility.ts

```diff
@@ -511,3 +514,9 @@ export function queryToBody(req: express.Request) {
     }
     return req;
 }
+
+export function mutable<T>(obj: T, targetid: number): Promise<T> {
+    if (targetid == config.global.defaultUserId)
```

Comment:
We should probably use `===` instead of `==`

Reply:
Will do